### ISSUE TITLE
Question re install instructions involving `prep.sh`

### ISSUE THREAD

**Issue by razlani** [linux-surface/surface-uefi-firmware#13]
I downloaded SurfaceBook2_Win10_18362_21.032.17273.0.msi and used the linux tool in package msitools to extract it but cannot find any directory/file information that corresponds to what you have in that diagram, ie 

└── surface-uefi-firmware
    ├── prep.sh
    ├── README.sh
    ├── repack.sh
    └── template.metainfo.xml

This bit.

Can someone kindly here explain in laymans terms how I can generate that info to pass into a repack?!  

I have the classic issue of trying to undervolt my surface book 2 and am prevented from doing so after the latest UEFI firmware.

**Comment by StollD** [comment]
The files in the diagram are the files from this repository. You dont have to extract the MSI yourself, just pass it to prep.sh / repack.sh and that will handle the extraction for you.

**Comment by razlani** [comment]
Thanks @StollD .. Just as I figured that out: It seems that I've encountered the same error as issue #12 :-(

Mine's a surface book 2/ubuntu
